src/common/plugin.ts

```diff
@@ -67,7 +71,7 @@ export abstract class Plugin implements IPlugin {
             : `one of the following channels: ${validChannelNames}.`
         }`;
       }
-    } else if (this.pluginCategoryName && this.pluginCategoryName !== categoryName) {
+    } else if (this.pluginCategoryName && this.pluginCategoryName.toLowerCase() !== categoryName) {
       return `Please use this command in the \`${this.pluginCategoryName}\` category.`;
```

Comment:
i dont think you need this branch anymore

Reply:
It's definitely necessary for plugins that define a category but no channels. The category can fail to match whether the plugin also has channels defined or not.